src/components/EventsLabel/EventsLabel.jsx

```diff
@@ -0,0 +1,23 @@
+import './EventsLabel.scss';
+
+import React, { PureComponent, PropTypes } from 'react';
+
+import cxHelpers from 'lib/decorators/classNameHelpers';
+import Icon from 'components/Icon';
+
+@cxHelpers("EventsLabel")
+class EventsLabel extends PureComponent {
+  static propTypes = {
```

Comment:
Is this like an empty method in Ruby that just needs to be defined for things to work?